### PR TITLE
Client extensions

### PR DESCRIPTION
**Pull request by ghost**
Relocated client extensions from a10-neutron-lbaas/a10-openstack

### CODE REVIEW

**Comment by Cedev** [comment]
```
   File "/usr/local/src/a10-neutron-lbaas/a10_neutron_lbaas/neutron_ext/common/constants.py", line 17, in <module>
     from a10_neutronclient.resources import a10_device_instance
 ImportError: cannot import name a10_device_instance
```

**Comment by Cedev** [comment]
```
  File "/usr/local/src/a10-neutronclient/a10_neutronclient/a10_device_instance.py", line 18, in <module>
    from a10_neutronclient.resources import a10_appliance
ImportError: cannot import name a10_appliance
```

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+mock>=1.0.1,<1.1.0
+nose
+nose-exclude
+
+# TODO(dougwig) -- can we remove this?
+-e git+https://github.com/openstack/neutron#egg=neutron
+
```
Comment (Cedev) [inline test-requirements.txt:7]: neutron shouldn't be a requirement for testing the client

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Many of our requirements have to be the same as the underlying neutron,
+# so while we will list the ones used explicity here, we will not necessarily
+# specifiy versions
+# a10-neutronclient
```
Comment (Cedev) [inline requirements.txt:4]: Where is `python-neutronclient`?

**Comment by ghost** [comment]
Closing because github has fallen on it's head?

**Pull request closed without merging**